### ISSUE TITLE
characterize RFI at nsfrb resolution for excision methods

### ISSUE THREAD

**Issue by mbsherma** [dsa110/dsa110-issues#272]

**Comment by caseyjlaw** [comment]
@mbsherma Are you satisfied with RFI flagging in the NSFRB system? If so, we can close. If not, we could update this issue to discuss what's needed.

**Comment by mbsherma** [comment]
@caseyjlaw Yes RFI flagging is sufficient and the classifier is able to reject RFI well. I'll go ahead and close
